app/views/pages/events.html.erb

```diff
@@ -1,69 +1,51 @@
 <section class="section">
-  <h1 class="title is-1">Events</h1>
+  <span class="events-page-wrapper">
+    <h1>View All Events</h1>
+    <p>
+      Find an event in your time zone and complete the challenge with others in your community! Or, check out our <a href="/eventkit">Hacktoberfest Event Organizer Kit</a> to create your own meetup.
+    </p>
+    <p>
+      <Button class="create-event" url="https://organize.mlh.io/host/hacktoberfest-2020">create new event</Button>
```

Comment:
```suggestion
      <a class="create-event" href="https://organize.mlh.io/host/hacktoberfest-2020">Create new event</a>
```
Use a link so it actually goes :)

Reply:
Good catch. I totally just copied that from the static site. It should be good now.